src/settings/MappingProfiles/MappingProfilesFormContainer/MappingProfilesFormContainer.js

```diff
@@ -1,35 +1,66 @@
-import React, { useState } from 'react';
-import { FormattedMessage } from 'react-intl';
+import React, {
+  useState,
+  useRef,
+} from 'react';
+import { useIntl } from 'react-intl';
+import { omit } from 'lodash';
 
-import { Layer } from '@folio/stripes/components';
+import {
+  Layer,
+  Callout,
+} from '@folio/stripes/components';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
 
 import { MappingProfilesTransformationsModal } from '../MappingProfilesTransformationsModal';
 import { MappingProfilesForm } from '../MappingProfilesForm';
 import { mappingProfileTransformations } from '../MappingProfilesTransformationsModal/TransformationsField/transformations';
 import { generateTransformationFieldsValues } from '../MappingProfilesTransformationsModal/TransformationsField';
 
 export const MappingProfilesFormContainer = props => {
+  const { onSubmit } = props;
   const [transformationModalOpen, setTransformationModalOpen] = useState(false);
+  const [selectedTransformations, setSelectedTransformations] = useState([]);
   const initialTransformationsValues = { transformations: generateTransformationFieldsValues(mappingProfileTransformations) };
+  const calloutRef = useRef(null);
+  const intl = useIntl();
 
   return (
-    <FormattedMessage id="ui-data-export.mappingProfiles.newProfile">
-      {contentLabel => (
-        <Layer
-          isOpen
-          contentLabel={contentLabel}
-        >
-          <MappingProfilesForm
-            {...props}
-            onAddTransformations={() => setTransformationModalOpen(true)}
-          />
-          <MappingProfilesTransformationsModal
-            isOpen={transformationModalOpen}
-            initialTransformationsValues={initialTransformationsValues}
-            onCancel={() => setTransformationModalOpen(false)}
-          />
-        </Layer>
-      )}
-    </FormattedMessage>
+    <Layer
+      isOpen
+      contentLabel={intl.formatMessage({ id: 'ui-data-export.mappingProfiles.newProfile' })}
+    >
+      <MappingProfilesForm
+        {...props}
+        transformations={selectedTransformations}
+        onSubmit={values => {
+          const mappingProfiles = { ...values };
+
+          mappingProfiles.transformations = selectedTransformations.map(transformation => omit(transformation, 'displayName'));
```

Comment:
Could you please clarify whether omit can be moved to the other omitted values?

https://github.com/folio-org/ui-data-export/pull/124/files#diff-e0dfc81555175ebb206276830471232dR17

Reply:
Yeah, this is a good point. I missed it during transition to the shared solution.